samples/drivers/adc/src/main.c

```diff
@@ -62,18 +75,16 @@ void main(void)
 			       adc_channels[i].dev->name,
 			       adc_channels[i].channel_id);
 
-			(void)adc_sequence_init_dt(&adc_channels[i], &sequence);
-
 			err = adc_read(adc_channels[i].dev, &sequence);
```

Comment:
Shouldn't this be moved outside the for loop? This would restrict `io-channels` to only contain channels from one device, but that might be ok?

Reply:
Same comment.

Reply:
Well, this commen ended up on the wrong line, I'll move it :)